workshop/content/docs/publish/vector.md

```diff
@@ -36,43 +32,84 @@ You are going to add a file `firenze-terrains.gpkg` to pygeoapi which is availab
     ``` {.yaml linenums="1"}
     firenze-terrains:
         type: collection 
-        title: 
-            it: Limiti amministrativi comunali ante 2014
-            en: Administrative boundaries before 2014
-        description: 
-            it: Limiti catastali (terreni) dal catasto. Agenzia del Territorio; SIT e Reti Informative;
-            en: Cadastral parcels (terrains) from the cadastre. Territory Agency; SIT and Information Networks;
+        title: Administrative boundaries before 2014
+        description: Cadastral parcels (terrains) from the cadastre. Territory Agency; SIT and Information Networks;
         keywords:  
             - Cadastral parcels
         links:
-            - type: text/html
-              rel: canonical  
-              title: 
-                it: Limiti amministrativi comunali ante 2014
-                en: Administrative boundaries before 2014
-              href: http://dati.cittametropolitana.fi.it/geonetwork/srv/metadata/cmfi:c539d359-4387-4f83-a6f4-cd546b3d8443
-              hreflang: it
+            -   type: text/html
+                rel: canonical  
+                title: Administrative boundaries before 2014
+                href: http://dati.cittametropolitana.fi.it/geonetwork/srv/metadata/cmfi:c539d359-4387-4f83-a6f4-cd546b3d8443
+                hreflang: it
         extents:
             spatial: 
                 bbox: [11.23,43.75,11.28,43.78] 
                 crs: http://www.opengis.net/def/crs/OGC/1.3/CRS84
         providers:
-            - type: feature
-              name: SQLiteGPKG
-              data: /data/firenze_terrains.gpkg # place correct path here
-              id_field: fid
-              table: firenze_terrains
+            -   type: feature
+                name: SQLiteGPKG
+                data: /data/firenze_terrains.gpkg # place correct path here
+                id_field: fid
+                title_field: codbo
+                table: firenze_terrains
     ```
 
-Save the file and restart the service. Navigate to localhost:5000/pygeoapi/collections to evaluate if the new dataset is available.
+Save the file and restart the docker compose. Navigate to localhost:5000/collections to evaluate if the new dataset is available.
 
 !!! tip 
 
     The sqlite driver incidentally has challenges to open the geopackage extension on MacOS. Read [this thread](https://docs.pygeoapi.io/en/latest/development.html#working-with-spatialite-on-osx) or try with an alternative data format.
  
+## pygeoapi as a WFS proxy
+
+An interesting use case for pygeoapi is to provide a OGC API Features interface over existing WFS endpoints. In this scenario you increase the usability of existing services to a wider audience. In this exersize we are setting op a API on top of an existing WFS hosted by the municipality of Florence.
```

Comment:
suggestion, add: "over existing WFS endpoints and even over ESRI FeatureServer APIs."